release/branch/cloudbuild.yaml

```diff
@@ -13,73 +13,214 @@
 # limitations under the License.
 
```

Comment:
Can you update the PR template and CONTRIBUTING.md?

Reply:
We can do that when we switch to serve the site from the new branch. Until then, we should use the same process as before.